packages/components/src/components/button/button_loading.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { HTMLProps } from 'react';
+import Loading from '../loading/loading';
+import '../loading/loading.scss';
+
+type TButtonLoadingProps = HTMLProps<HTMLDivElement>;
+
+const ButtonLoading = (props: TButtonLoadingProps) => {
```

Comment:
```suggestion
import Loading, { TLoadingProps } from '../loading/loading';
import '../loading/loading.scss';

const ButtonLoading = (props: TLoadingProps) => {
```
@jim-deriv ButtonLoading is passing all its props to its child, thus it expects the same props🙂